testsite/myapp/settings.py

```diff
@@ -115,7 +116,8 @@
 )
 
 CIVET_SASS_ARGUMENTS = ('--compass', )
-CIVET_IGNORE = ['bower_components', 'to_be_ignored']
+CIVET_IGNORE_PATTERNS = ['']
```

Comment:
couldn't think of any good things to filter.  Sass files aren't actually currently filterable due to `get_shortest_topmost_directories`.  I tested `coffee` in `CIVET_IGNORE_PATTERNS` and Civet successfully skipped over coffeescript files though, so there's that.